frontend/src/app/modules/settings/team/team-members/team-members.component.ts

```diff
@@ -161,29 +163,37 @@ export class TeamMembersComponent extends BaseComponent implements OnInit, OnDes
             );
     }
 
-    changeTeamMemberRole(user: IUser, newRole: Role) {
+    changeTeamMemberRole(user: ITeamMember, newRole: Role) {
         const teamMember: ITeamMember = {
             id: user.id,
             image: user.image,
-            name: user.userName,
+            name: user.name,
             email: user.email,
             pageLink: '',
             role: newRole,
             status: Status.Pending,
         };
 
-        this.teamService
-            .updateTeamMember(teamMember)
-            .pipe(this.untilThis)
-            .subscribe(
-                () => {
-                    this.notificationService.showSuccessMessage('Team member`s role was successfully changed');
-                    this.reloadTeamMembers();
-                },
-                (error) => {
-                    this.notificationService.showErrorMessage(error);
-                },
-            );
+        if (newRole === Role.Owner && this.ownerId) {
+            this.notificationService.showErrorMessage('Team member with owner role should be the only');
+            this.reloadTeamMembers();
+        } else if (user.id === this.ownerId) {
+            this.notificationService.showErrorMessage('Owner role can be modified only when owner leave the team');
+            this.reloadTeamMembers();
```

Comment:
Same here

Reply:
If I try to change role, I will see error but select has new value (only visually). We need reload members or reload page.
![image](https://user-images.githubusercontent.com/71783287/189725153-96913514-3264-40bc-a06e-7059c68c1c2e.png)

Reply:
For example, I can change this duplicate code to the end of method.
```suggestion
            this.reloadTeamMembers();
```

Reply:
I think resetting the value in the select will be better than an api request